src/test/addingToFavourites.test.tsx

```diff
@@ -1,63 +1,69 @@
+import "@testing-library/jest-dom";
 import { fireEvent, screen, waitFor } from "@testing-library/react";
+import BookCard from "components/BookCard";
+import type { Book } from "src/types";
 import { render } from "test/test-utils";
-import "@testing-library/jest-dom";
-import Book from "components/Book";
 
-const dummyBook = {
+const dummyBook: Book = {
   title: "The Great Gatsby",
   authors: [{ name: "F. Scott Fitzgerald" }],
   bookshelves: ["Best Books Ever"],
-  formats: {},
+  formats: {
+    "text/html": "https://www.gutenberg.org/ebooks/64317",
+    "image/jpeg":
+      "https://www.gutenberg.org/cache/epub/64317/pg64317.cover.medium.jpg",
+  },
   languages: ["en"],
   subjects: ["fiction"],
   translators: [],
   id: 5,
 };
```

Comment:
raczej bym użył tutaj satisfies

https://www.totaltypescript.com/how-to-use-satisfies-operator

```suggestion
const dummyBook = {
  title: "The Great Gatsby",
  authors: [{ name: "F. Scott Fitzgerald" }],
  bookshelves: ["Best Books Ever"],
  formats: {
    "text/html": "https://www.gutenberg.org/ebooks/64317",
    "image/jpeg":
      "https://www.gutenberg.org/cache/epub/64317/pg64317.cover.medium.jpg",
  },
  languages: ["en"],
  subjects: ["fiction"],
  translators: [],
  id: 5,
} satisfies Book;
```